api.planx.uk/send/bops.ts

```diff
@@ -51,6 +48,8 @@ const sendToBOPS = async (req: Request, res: Response, next: NextFunction) => {
   const domain = `https://${req.params.localAuthority}.${process.env.BOPS_API_ROOT_DOMAIN}`;
```

Comment:
Yes - will address in submissions api ticket! Will have to think about generic solution that also accounts for uniform's sesionId -> passport -> `property.localAuthorityDistrict` instead of flow -> team !